api/trip/serializers.py

```diff
@@ -20,19 +20,14 @@ class StateSerializer(serializers.ModelSerializer):
     class Meta:
         model = State
         fields = '__all__'
-
-class LocationSerializer(serializers.ModelSerializer):
-    class Meta:
-        model = Location
-        fields = '__all__'
     
 class TripSerializer(serializers.ModelSerializer):
     locations = LocationSerializer(many=True, read_only=True)
     activities = ActivitySerializer(many=True, read_only=True)
     # Not sure is need populate the attendees' information or not,
     # Because some info maybe should not show to others.
     # Currently, It just populate the attendees' all information
-    attendees = UserSerializer(many=True, read_only=True)
+    # attendees = UserSerializer(many=True, read_only=True)
```

Comment:
Is this line of code needed at all? If not, could you just remove it all together rather than commenting it out?

Reply:
It doesn't need. Sorry about this. I already removed it and fixed the conflicts.